bs-react-native-next/bsconfig.json

```diff
@@ -16,6 +16,9 @@
   "ppx-flags": [
     "./node_modules/reason-react/_build/default/jsx_ppx/reactjs_jsx_ppx_v3.exe"
   ],
-  "bsc-flags": ["-bs-super-errors", "-warn-error @a", "-w -41..42-102"],
```

Comment:
`-bs-super-errors` is the default now AFAIK, so I removed it. I added `-bs-no-version-header` to get rid of the version headers in future diffs.

Reply:
Make sense!